client/src/pages/Editor.tsx

```diff
@@ -0,0 +1,236 @@
+import { useEffect, useRef, useMemo, useState, useCallback } from 'react';
+import { Button } from 'react-bootstrap';
+import { ActionExport, DrawIoEmbed, DrawIoEmbedRef} from 'react-drawio';
+import { useLocation } from 'react-router-dom';
+// import { redirect } from 'react-router-dom';
+
+
+const CONFIG = {
+    defaultEdgeStyle: {endSize: 12, startSize: 12},
+    // autosaveDelay: 10000, // 10 seconds
```

Comment:
Intentional to have autosave commented as well?

Reply:
yes